src/build_queue.rs

```diff
@@ -206,7 +206,7 @@ impl BuildQueue {
                 to_process.name, to_process.version
             )
         });
-        self.metrics.total_builds.inc();
+        self.metrics.instance.total_builds.inc();
```

Comment:
Should we change these to take `InstanceMetrics` instead of `Metrics`? (Probably by moving the `Arc` inside the `Metrics` so each is shared individually).

Reply:
Design-wise you're right, though this then would need this split across all usages of metrics. 

I'll give it a try, and see how much breaks

Reply:
I updated the code with what I think you meant, thanks for the push!